src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setColor(color.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
there will be created several instances of ColorSupplier and Random, lets just make these class-level variables

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/81781479/204826995-27b65e71-b4c0-448a-9a15-d40f1cea1bf5.png">